### ISSUE TITLE
Attacked after all characters were dead

### ISSUE THREAD

**Issue by Ink230** [Legend-of-Dragoon-Modding/Severed-Chains#614]
Dramod enabled, HARD MODE

Setup as pictured. Facing Greham and Feyrbrand. 

Feyrbrand attacked, then Lavtiz was poisoned and died to poison damage on his turn. Then Greham got another turn to attack despite everyone being dead.

![image](https://github.com/Legend-of-Dragoon-Modding/Legend-of-Dragoon-Java/assets/29797557/6861fc23-2469-4958-b4f4-8adde2f58815)

**Comment by Lothar635** [comment]
could this be a mod issue not SC?

**Comment by LordMonoxide** [comment]
I don't think so. We've had a couple of crazy things like this happen, but it's so rare it's hard to make any conclusions about it. We did some group investigation a while ago and it seemed like it could have something to do with an enemy doing _exactly_ the right about of damage to kill you.

**Comment by avionanx** [comment]
Need to look into status interactions, especially stun into poison

**Comment by avionanx** [comment]
Cannot reproduce, closed until someone triggers it
